src/lib/datepicker/calendar.ts

```diff
@@ -48,6 +69,13 @@ import {MatYearView} from './year-view';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class MatCalendar<D> implements AfterContentInit, OnDestroy, OnChanges {
+
```

Comment:
nit: no blank line

Reply:
done in a88fecbec2c3292017613bf4e75c38ff7c632042